docs/reference/ingest.asciidoc

```diff
@@ -539,6 +540,29 @@ PUT _ingest/pipeline/my-pipeline
 Use a Mustache template snippet to access metadata field values. For example,
 `{{{_routing}}}` retrieves a document's routing value.
 
+[source,console]
+----
+PUT _ingest/pipeline/my-pipeline
+{
+  "processors": [
+    {
+      "set": {
+        "description": "Use geo_point dynamic template for address field",
+        "field": "_dynamic_templates",
+        "value": {
+          "address": "geo_point"       
+        }
+      }
+    }
+  ]
+}
+----
+
+The set processor above tells ES to use the dynamic template named `geo_point`
+for the field `address` (if this field does not exist in the index). Note that 
+this processor overrides only the `_dynamic_templates` of the field `address`
+but leaves the settings of other fields intact.
```

Comment:
I think that we should try and clarify this sentence: "this processor overrides the dynamic template for the field `address` if already defined in the bulk request, but has no effect on other dynamic templates defined in the bulk request."

Reply:
fixed in https://github.com/elastic/elasticsearch/pull/71716/commits/6245e1e8e276413ce968daaa3a2f25873c03c92b